src/fmu/sumo/uploader/_sumofile.py

```diff
@@ -151,8 +159,22 @@ def upload_to_sumo(self, sumo_parent_id, sumo_connection):
                 }
             )
             pass
+        except Exception as err:
+            logger.warn(f"Metadata upload exception {err} {type(err)}")
+            result.update(
+                {
+                    "status": "failed",
+                    "metadata_upload_response_status_code": 500,
+                    "metadata_upload_response_text": str(err),
```

Comment:
Burde denne bruke `err.response.status_code ` i stedet for 500?

Reply:
Could be any kind of exception here, even without .response